README.md

```diff
@@ -12,6 +12,10 @@ Join the [SFXD Discord Community](https://discordapp.com/invite/JG4Bc4q)!
 
 # Community Repos
 
+- [Check if JS Lib compatable] (https://github.com/JesseGlover/Test-LWC-and-JS-Libraries/tree/master)
```

Comment:
This library doesn't look like it actually automates and tests anything, I think we would be better with just a list in the readme of compatible libraries.